apple/RNCWebView.m

```diff
@@ -311,9 +311,7 @@ - (void)didMoveToWindow
     [_webView addObserver:self forKeyPath:@"estimatedProgress" options:NSKeyValueObservingOptionOld | NSKeyValueObservingOptionNew context:nil];
     _webView.allowsBackForwardNavigationGestures = _allowsBackForwardNavigationGestures;
 
-    if (_userAgent) {
-      _webView.customUserAgent = _userAgent;
-    }
+    _webView.customUserAgent = _userAgent;
```

Comment:
Removed unnecessary branch. It's perfectly valid to set customUserAgent to `nil`.

Reply:
If you set it to `nil` does it leave the default user-agent in place?

Reply:
`nil` is the default value of this property. I tested this and setting to `nil` restores the default user agent.